pkg/scalers/kafka_scaler.go

```diff
@@ -163,6 +164,18 @@ func parseKafkaAuthParams(config *ScalerConfig, meta *kafkaMetadata) error {
 					return errors.New("no oauth token endpoint uri given")
 				}
 				meta.oauthTokenEndpointURI = strings.TrimSpace(config.AuthParams["oauthTokenEndpointUri"])
+
+				meta.extensions = make(map[string]string)
+				for _, extension := range strings.Split(config.AuthParams["extensions"], ",") {
```

Comment:
Maybe this is super obvious but I'm a noobie with Kafka, are these extensions some kind of secret? If there aren't, we shouldn't read them from AuthParams

Reply:
I think in most cases these extensions should not be sensitive. I put them next to the other oAuth options to keep things consistent.
Should I add them to the metadata of the ScaledObject instead?
`tls` and `sasl` can be defined both in the ScaledObject or in AuthParams? Should I do the same for those extensions?
If the configuration is not done in the AuthParams the param should maybe be renamed to oauthExtensions?

Reply:
The problem is that `AuthParams` is for sensitive data semantically and we try to keep it for that. I prefer to move it to metadata is it's not a secret. @kedacore/keda-contributors ?

Reply:
I adjusted it now. I will update the docs, if we have decided that this format is ok.